src/main/java/ml/shifu/shifu/core/processor/TrainModelProcessor.java

```diff
@@ -628,7 +628,7 @@ protected int runDistributedTrain() throws IOException, InterruptedException, Cl
 
             if(isParallel) {
                 TailThread tailThread = startTailThread(progressLogList.toArray(new String[0]));
-                status += guaguaClient.run();
+                guaguaClient.run();
```

Comment:
could you revert such line back? This is changed by Zhanghao to track job status well. I need upgrade guagua for a release version to make you guys no need install Guagua